src/Cli/dotnet/Commands/Run/VirtualProjectBuildingCommand.cs

```diff
@@ -160,13 +177,37 @@ public ImmutableArray<CSharpDirective> Directives
 
     public override int Execute()
     {
+        bool msbuildGet = MSBuildArgs.GetProperty is [_, ..] || MSBuildArgs.GetItem is [_, ..] || MSBuildArgs.GetTargetResult is [_, ..];
+        bool evalOnly = msbuildGet && RequestedTargets is null or [];
+        bool minimizeStdOut = msbuildGet && MSBuildArgs.GetResultOutputFile is null or [];
+
         var verbosity = MSBuildArgs.Verbosity ?? VerbosityOptions.quiet;
-        var consoleLogger = TerminalLogger.CreateTerminalOrConsoleLogger([$"--verbosity:{verbosity}", .. MSBuildArgs.OtherMSBuildArgs]);
+        var consoleLogger = minimizeStdOut
+            ? new SimpleErrorLogger()
```

Comment:
the rest of this looks good, but I don't actually know what the behavior of the SimpleErrorLogger is. What does this look like in the case of errors?

Reply:
It writes each error on a single line something like this: